app/screens/MixingElixir/MixingElixirSuccess/MixingElixirSuccess.tsx

```diff
@@ -90,7 +92,8 @@ export const MixingElixirSuccessScreen: React.FC<IMixingElixirSuccessScreenProps
         default:
           return PLANTS_SVG.courageGrown;
       }
-    }, [currentPlant]);
+      // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
```

Comment:
Why should we use it with `useMemo` and disable eslint?
just use `useRef` once at start and that's it